### PR TITLE
Redesign startup info in command line and add projects path

### PR DESCRIPTION
**Pull request by J535D165**
Screenshot 2024-01-26 at 11 58 22

- New table for relevant info
- Add projects location
- Add backup instructions

The docs and discussion platform need to be updated around the release of this feature. https://github.com/asreview/asreview/discussions/1203

This PR displays a non-default location for the projects, which will be part of another PR. The location will be more platform-dependent in that version (but technically more correct and better backup functionalities).

### CODE REVIEW

**Comment by J535D165** [comment]
image

Resolved some issues on CMD.exe, hopefully.

File: asreview/webapp/entry_points/lab.py [hunk 1]
```diff
@@ -132,15 +117,50 @@ def lab_entry_point(argv):
...
+            "Run `pip install --upgrade asreview` to update."
+        )
+
+    console.print(
+        "\n\nMake regular backups of the ASReview"
+        " projects folder to prevent data loss."
+    )
     if not args.no_browser:
         _open_browser(start_url)
+        console.print(f"If your browser doesn't open, navigate to {start_url}.\n\n\n")
```
Comment (Rensvandeschoot) [inline asreview/webapp/entry_points/lab.py:156]:
```suggestion
        console.print(f"\n\nIf your browser doesn't open, navigate to {start_url}.\n\n\n")
```

Comment (Rensvandeschoot) [reply asreview/webapp/entry_points/lab.py:156]: I propose an extra line to make sure the note on back-up is noted

**Review by Rensvandeschoot — APPROVED** [review]

File: asreview/webapp/entry_points/lab.py (hunk 1 above)
Comment (J535D165) [inline asreview/webapp/entry_points/lab.py:156]:
```suggestion
        console.print(f"\nIf your browser doesn't open, navigate to {start_url}.\n\n\n")
```

**Pull request merged**